### ISSUE TITLE
GraphQL Review Team initiative.

### ISSUE THREAD

**Issue by IvanGoncharov** [graphql/graphql-js#3382]
As discussed on `graphql-js-wg`, we decided to run the "GraphQL Review Team" initiative.

## Goal

Previously, we had a lot of PRs that stuck for months (sometimes for years) without any feedback.
The goal of this initiative is to provide timely feedback on PRs, especially from new contributors.

## Scope

This initiative is an experiment to see if this model will work or not.
Moreover, it is intended to be a temporary measure until we get a team of active core maintainers.

## Expectation

Members of @graphql/graphql-js-reviewers are expected to provide feedback on PR once tagged (by author, core member, etc.).
They also are encouraged to review any PRs at any time, based on their individual time commitments. 

Since team members are volunteers, please value their time and tag them only if:
1. PR didn't get any feedback in 48 hours after it was posted.
2. All previous feedback was addressed, and PR is ready to merge.

## Merge rule

PRs can be merged as-is following the current procedure.
However, in case if PR (that does not require changes to the GraphQL spec) is stuck, anyone who has commit rights to this repo can merge it if all the below requirements are satisfied:
1. At least two members of the review team approved this PR, and no one rejected it. 
2. Two weeks have passed since the last non-trivial change was added, and at least a week has passed since all review comments were resolved.
3. All review comments are addressed (we have a CI rule for that)
4. All non-trivial changes invalidate PR approval. Please re-request review in that case.

All PRs related to GraphQL spec proposal (typically have [`spec RFC`](https://github.com/graphql/graphql-js/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc+label%3A%22spec+RFC%22) label)
should be reviewed extremely carefully by members of https://github.com/graphql/graphql-wg and thus excluded from the above rule.

## Team 

*Updated on 26-01-2022*

* @martijnwalraven 
* @Cito
* @yaacovCR 
* @IvanGoncharov 
* @saihaj

**Comment by hwillson** [comment]
@IvanGoncharov just a heads up that [@graphql/graphql-js-reviewers](https://github.com/orgs/graphql/teams/graphql-js-reviewers) isn't publicly viewable.

**Comment by dotansimha** [comment]
@IvanGoncharov can we make https://github.com/orgs/graphql/teams/graphql-js-reviewers public?

**Comment by saihaj** [comment]
I think GitHub doesn't have a way.  There are only 2 options and a visible means they are visible to members of organizations. I don't think there is a way to make it public for people who are not part of the organization. https://docs.github.com/en/organizations/organizing-members-into-teams/about-teams#team-visibility

**Comment by n1ru4l** [comment]
Could we just set up a document within this repository that lists the members, or list them in this issue, which is already pinned?

**Comment by IvanGoncharov** [comment]
@n1ru4l Great idea!
Added list in the top comment.

**Comment by IvanGoncharov** [comment]
I'm in Ukraine🇺🇦 right now.
The situation is stable and I managed to evacuate my relatives to the western part of Ukraine.
Internet is more or less stable and I returning back to reviewing PRs and triaging issues.

However, there is a possibility I will be drafted or I would be unable to work for some other reasons.
If something like that happens, please use [review team](https://github.com/graphql/graphql-js/issues/3382#issue-1062597594) to unblock PRs and for general maintenance.
I have full trust in members of the review team to do this:
@martijnwalraven
@Cito
@yaacovCR
@saihaj

If you will be blocked by something (GH admin rights, graphql.org access, etc.) please reach TSC: 
https://github.com/graphql/graphql-wg/blob/main/GraphQL-TSC.md#tsc-members-1

**Comment by benjie** [comment]
I'm glad you're safe @IvanGoncharov and thanks for everything you've done for GraphQL.js over the years! Your impact has been massive, and you have left some very large shoes to fill!

Please do reach out to other potential reviewers - more hands would be welcome I think.

@IvanGoncharov I see from the [CONTRIBUTING document instructions on publishing to npm](https://github.com/graphql/graphql-js/blob/main/.github/CONTRIBUTING.md#release-on-npm); but do you know who currently has access to do so? Who can add more people to that list? (I don't think I am on it, my npm username is `benjie` if someone can add me as an admin or whatever.)

I don't have the bandwidth to be able to take on review duties/etc at the moment, but I'd be happy to get the GraphQL.js WG up and running again and facilitate the meetings until a more permanent host emerges. Alternatively if one of @graphql/graphql-js-reviewers would like to host the meetings I'd be more than happy to help them get set up; we have tools to help manage the agendas and what not now so things should be easier than they were a couple years ago.

As always, feel free to DM me on Discord if you need anything.

Thanks again, Ivan! :raised_hands:

**Comment by leebyron** [comment]
Thanks for everything you've done over the years as a lead maintainer for this most important project. Also, thank you for being proactive about finding a successor!

Tapping in @graphql/tsc for awareness and support to help

**Comment by leebyron** [comment]
Should we open a dedicated issue to discuss review team change?

**Comment by JoviDeCroock** [comment]
Hey,

I'd love to volunteer in helping out both in reviews as well as implementation

**Comment by bignimbus** [comment]
The Apollo Client maintainers are also interested in helping out.  We have a in-person get-together that would likely prevent us from attending the 28 Feb WG meeting but are looking forward to contributing 🙏🏻

**Comment by benjie** [comment]
@bignimbus Awesome! Is this a one-time scheduling conflict or liable to be more common? I just created the event based on the old schedule but would be happy to shift it around to accomodate people's availability.

**Comment by bignimbus** [comment]
It's a one-time conflict

**Comment by benjie** [comment]
We'll stick with the current schedule for now then, and look forward to seeing your team members at future WGs!

**Comment by yaacovCR** [comment]
Always open to more people being involved at all levels. Just closing this issue in the name of house keeping. One way to get involved is to join a wg meeting: https://github.com/graphql/graphql-js-wg
